pom.xml

```diff
@@ -76,6 +81,29 @@
                 <artifactId>jansi</artifactId>
                 <version>${version.org.fusesource.jansi}</version>
             </dependency>
+            <!-- Arquillian container test SPI for extension -->
+            <dependency>
+                <groupId>org.jboss.arquillian.container</groupId>
+                <artifactId>arquillian-container-test-spi</artifactId>
+                <version>${version.org.jboss.arquillian.container.test.spi}</version>
```

Comment:
This must have the same version as arquillian thus use `version.org.jboss.arquillian` property here.

Reply:
Done.